apps/portal/src/components/pages/RecommendationsPage.js

```diff
@@ -315,17 +315,29 @@ const RecommendationsPage = () => {
         };
     }, []);
 
-    const heading = pageData && pageData.signup ? t('Welcome to {{siteTitle}}', {siteTitle: title, interpolation: {escapeValue: false}}) : t('Recommendations');
-    const subheading = pageData && pageData.signup ? t('Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.') : t('Here are a few other sites you may enjoy.');
-
-    if (!recommendationsEnabled) {
-        return null;
-    }
-
     if (recommendations === null) {
         return <LoadingPage/>;
     }
 
+    const heading = pageData && pageData.signup ? t('Welcome to {{siteTitle}}', {siteTitle: title, interpolation: {escapeValue: false}}) : t('Recommendations');
+
+    /* Possible cases: 
+    - no recommendations found - subhead says no recommendations are available.
+    - recommendations found - show generic message
+    - recommendations found and user just signed up - show specific message
+    */
+   
+    let subheading;
+    if (recommendationsEnabled && recommendations && recommendations.length > 0) {
```

Comment:
Sorry, one more thing @cathysarisky would we be better initialising recommendations as `[]` instead of `null`? Then we can just check on `.length` to know if there are any recommendations or not 🤔 (not sure if it explicitly needs to be `null`)

Reply:
It was that way when I got to it, but I agree, it'd be a little simpler if it were initialized as [].  I can push that change over?

Reply:
Oh, no, hang on.  Line 318. We're checking if recommendtions is null and serving the loading page if it is.  So we do need to distinguish between null/uninitialized vs empty array because there are no recs, I think.